packages/bundle-source/src/is-entrypoint.js

```diff
@@ -3,7 +3,10 @@ import url from 'url';
 import process from 'process';
 import fs from 'fs';
 
+const favor = (primary, secondary) =>
+  typeof primary === 'undefined' ? secondary : primary;
```

Comment:
```suggestion
  primary === undefined ? secondary : primary;
```

Since we statically see that `primary` is a variable in lexical scope, your code and my suggestion must be equivalent. The only reason to use the `typeof name === 'undefined'` instead is if `name` might not be a variable name in scope and we wish to avoid throwing a `ReferenceError`.